requirements.txt

```diff
@@ -8,8 +7,7 @@ canonicalwebteam.http==1.0.1
 canonicalwebteam.versioned_static==1.0.2
 canonicalwebteam.yaml-responses[django]==1.1.0
 whitenoise==4.1.2
-statsd==3.3.0
-talisker[gunicorn]==0.14.3
+talisker[gunicorn,prometheus,raven,django]==0.14.3
```

Comment:
What is Raven again? Why do we need it?

If we need to specify "django" here, how has it been working up until now?

Reply:
raven is the library for sentry. We need it to have well configured sentry messages

adding django is useful for us to get `talisker.django.middleware` that adds the correct values in the prometheus metrics for example.